docassemble/CLAGuardianship/data/questions/petition_for_removal_of_guardian.yml

```
@@ -506,6 +509,15 @@ code: |
 code: |
   user_ask_role = "plaintiff"
 ---
+code: |
+  motion_temporary_guardian_attachment.enabled = False
+  
+  if requested_guardians.there_are_any:
+    if requested_temporary_guardians.there_are_any:
+      motion_temporary_guardian_attachment.enabled = True 
+  
+  set_temp_guardian_doc = True
```

Just want to make sure that the default value isn't triggered before the rest of the block executes, so I'm adding the explicit default options in the logic.

```suggestion
  
  if requested_guardians.there_are_any:
    if requested_temporary_guardians.there_are_any:
      motion_temporary_guardian_attachment.enabled = True 
    else:
      motion_temporary_guardian_attachment.enabled = False
  else:
    motion_temporary_guardian_attachment.enabled = False
  
  set_temp_guardian_doc = True
```

---

I don't think the `set_temp_guardian_doc` is necessary either--usually I let Docassemble find the block on its own by triggering the undefined variable, in this case, `motion_temporary_guardian_attachment.enabled`. But I'll leave this alone.